app/views/admin/books/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2><%= t('books.edit.header') %> </h2>
+
+<%= simple_form_for @book, url: admin_book_path(@book) do |form| %>
+  <%= form.input :title, label: t('simple_form.labels.title') %>
+  <%= form.input :author, label: t('simple_form.labels.author') %>
+  <%= form.submit %>
```

Comment:
You can extract these form fields to a partial so that you don't need to duplicate them between the edit and new view